.github/workflows/lint.yaml

```diff
@@ -1,46 +1,56 @@
 name: "Code Quality"
+
 on:
   workflow_dispatch:
   push:
     branches:
       - main
-    paths-ignore:
-      - 'README.md'
-      - 'docs/**'
   pull_request:
-    paths-ignore:
-      - 'README.md'
-      - 'docs/**'
 
 permissions:
   contents: read
   pull-requests: read
+
 jobs:
   lint:
     runs-on: "ubuntu-latest"
     steps:
       - uses: actions/checkout@v4
-        with:
-          ref: ${{ github.event.pull_request.head.sha }}
-      - name: Checkout
-        run: |
-          commits=${{ github.event.pull_request.commits }}
-          if [[ -n "$commits" ]]; then
-            # Prepare enough depth for diffs with master
-            git fetch --depth="$(( commits + 1 ))"
-          fi
-
-      - name: Check commit message
-        run: hack/check-commits.sh
-
-      - name: Setup all dependencies
-        uses: ./.github/actions/setup-dependencies
-
-      - name: Check that generated files have been updated and that go.mod is tidy
-        run: |
-          make generate
-          go mod tidy -v
-          git diff --exit-code
+      #   with:
```

Comment:
I guess it's WIP yet, right? :-)

Reply:
Only until I can find out why these checks fail even when I revert them to original. grrrr

Reply:
Turns out golangci-lint v1.54 does not like golang 1.23 that we're using in some CI tests, although our artifacts correctly build with 1.21. I've created PR #593 to fix that and rebased this PR on it.